src/components/landingPage/Navbar.js

```diff
@@ -1,35 +1,93 @@
-import React from 'react';
 import { NavLink } from 'react-router-dom';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import img from '../../assets/logo.png';
 
-const Navbar = () => (
-  <nav className="navbar fixed-top navbar-expand-lg nav-bg">
-    <img src={img} className="image-fluid" alt="" />
-    <h6 className="navbar-brand col-sm-10" href="#">Authors Haven</h6>
-    <button
-      className="navbar-toggler  navbar-light"
-      type="button"
-      data-toggle="collapse"
-      data-target="#navbarSupportedContent"
-      aria-controls="navbarSupportedContent"
-      aria-expanded="false"
-      aria-label="Toggle navigation"
-    >
-      <span className="navbar-toggler-icon" />
-    </button>
-    <div className="collapse navbar-collapse" id="navbarSupportedContent">
-      <ul className="navbar-nav mr-auto">
-        <li className="nav-item active">
-          <NavLink className="btn ah-btn ah-btn-nav js-scroll-trigger nav-link nav-text" to="signup">
-            Signup
-          </NavLink>
-        </li>
-        <li className="nav-item">
-          <NavLink className="btn ah-btn ah-btn-nav js-scroll-trigger nav-link nav-text" to="login">Login</NavLink>
-        </li>
-      </ul>
-    </div>
-  </nav>
-);
+export class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoggedIn: false,
+    };
+  }
 
-export default Navbar;
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isLoginSuccess === true) {
+      this.state.isLoggedIn = true;
+    }
+  }
+
+  render() {
+    const username = localStorage.getItem('username');
+    const { isLoggedIn } = this.state;
+    return (
+      <nav className="navbar fixed-top navbar-expand-lg nav-bg">
+        <div className="container">
+          <img src={img} className="image-fluid" alt="" />
```

Comment:
Good work with the  Navbar. However, consider adding text in the `alt' if an image is not loaded

Reply:
An `alt` for the logo has been added